### PR TITLE
 Not possible to decode resources while decompiling Facebook lasted APK

### PR DESCRIPTION
**Pull request by ldavidsp**
Signed-off-by: Luis Solorzano <luisnic20@gmail.com>

### CODE REVIEW

**Comment by ldavidsp** [comment]
Working well for me.

**Comment by ldavidsp** [comment]
@iBotPeaches

**Comment by iBotPeaches** [comment]
Thanks for the PR! It looks like Windows broke, but I'll fix that tonight then ask you to rebase tomorrow.

**Comment by iBotPeaches** [comment]
If you rebase this, it should be good to go.

**Comment by iBotPeaches** [comment]
Merged master in. Awaiting build status.

File: brut.apktool/apktool-lib/src/main/java/brut/androlib/res/data/value/ResValueFactory.java [hunk 1]
```diff
@@ -108,7 +108,7 @@ public ResBagValue bagFactory(int parent, Duo<Integer, ResScalarValue>[] items,
             return new ResPluralsValue(parentVal, items);
         }
 
-        if (ResTypeSpec.RES_TYPE_NAME_STYLES.equals(resTypeName)) {
+        if (resTypeName.equals(resTypeName)) {
```
Comment (iBotPeaches) [inline brut.apktool/apktool-lib/src/main/java/brut/androlib/res/data/value/ResValueFactory.java]: Not sure about this. This makes this 100% true every-time no? So what about the ResAttr detection below this? What it looks like you are trying to do is default an unknown res type to styles, but not sure if that is the correct go here.

**Comment by iBotPeaches** [comment]
Thanks for the PR, but not going to move forward with this. I don't suspect this is the proper way to resolve this.

**Pull request closed without merging**